pkg/traceroute/icmp6.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package traceroute
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"golang.org/x/net/ipv6"
+)
+
+func (t *Trace) SendTracesICMP6() {
+	conn, err := net.ListenPacket("ip6:ipv6-icmp", t.SrcIP.String())
+	if err != nil {
+		fmt.Println(err)
```

Comment:
it's a bit inconsistent: in some places you log.Fatal and here fmt.Println(err), but printing error and continue not really make sense here?

Reply:
Agree. Need to clean that up or at least make it consistent